tests/test_clusterv2.py

```diff
@@ -0,0 +1,362 @@
+from __future__ import print_function
+import sys
+import os
+if sys.version_info > (2, 7, 0):
+    import unittest
+else:
+    import unittest2 as unittest
+from mock import Mock
+import tempfile
+sys.path.append(os.path.join(os.path.dirname(__file__), '../bin'))
+import qds
+from qds_sdk.connection import Connection
+from test_base import print_command
+from test_base import QdsCliTestCase
+from qds_sdk.cloud.cloud import Cloud
+
+class TestClusterCreate(QdsCliTestCase):
```

Comment:
Other than cluster clone , create and update there are no code changes . For other cluster apis we don't use version. Even if the the user pass v2 or v1.3 it will use v1.2 only.